package.json

```diff
@@ -12,8 +12,8 @@
   },
   "dependencies": {
     "aws-sdk": "latest",
-    "npm": "8.6.0",
-    "yargs": "17.4.0"
+    "npm": "8.19.4",
```

Comment:
Latest version is `9.8.1` - any chance we could that?

Reply:
Maybe. Only reason I didn't go that high is I don't know how thorough the tests are, so didn't want to jump a major version.
Thoughts @Zordrak ?

Reply:
Is there a reason to depend on a specific version rather than using a carat or a greater or equal? It feels like given this is included in other node projects it should not be explicit when it does not need to be?